plugins/woocommerce/src/Logging/RemoteLogger.php

```diff
@@ -1,6 +1,8 @@
 <?php
```

Comment:
Since the class name has the Logger suffix and most of the other directories are in plural form, how about naming the directory Loggers?

New structure:
```
Loggers
	RemoteLogger
		- RemoteLoggerRelatedClassA.php
		- RemoteLoggerRelatedClassA.php
	RemoteLogger.php
```

Reply:
Sounds good. I'll make the change. 👍

Reply:
Ok, done! 9d7d391a983edb71d20364e9b770dc23f686c707

Reply:
FWIW, I think `Logging` was a good choice as we already have `Internal\Admin\Logging` where other logging classes live. But this is obviously not a blocker, and being in the internal namespace, we could rename it later if necessary.

Reply:
Thanks for raising this! I didn't put it in the existing Logging namespace because we don't only use it in the admin area, so I created a new namespace. I'm happy to move it if you think it's better to keep it all together. 🙂

Reply:
Ah, got it. I see what you mean now. Thanks for clarifying. That makes sense. I'll make the change in this PR since this is a very minor change. 🙂